src/main/java/com/minecolonies/coremod/inventory/ContainerRack.java

```diff
@@ -17,6 +18,7 @@
 /**
  * The container class for the rack.
  */
+@ChestContainer
 public class ContainerRack extends net.minecraft.inventory.Container
```

Comment:
Did you produce a jar and tested it with and without invtweaks installed?

Reply:
I did and no errors as far as I know

Reply:
okay, sounds good

Reply:
This will not work if InventoryTweaks is not available as far as i know.
@ChestContainer still needs to be loadable in this case...... Please verify this again!.

Reply:
I agree with @OrionDevelopment here, don't think this would work